test/adaptors/postgres-articles.test.ts

```diff
@@ -0,0 +1,143 @@
+import { literal, namedNode, quad } from '@rdfjs/data-model';
+import all from 'it-all';
+import 'jest-rdf';
+import { DatasetCore, NamedNode } from 'rdf-js';
+import pgPromise, { IBaseProtocol, IMain } from 'pg-promise';
+import PostgresArticles from '../../src/adaptors/postgres-articles';
+import ArticleNotFound from '../../src/errors/article-not-found';
+import NotAnArticle from '../../src/errors/not-an-article';
+import { schema } from '../../src/namespaces';
+import createArticle from '../create-article';
+import db from '../../src/db';
+import dataFactory from '../../src/data-factory';
+
+let postgresPromise: IMain;
+let database: IBaseProtocol<IMain>;
+
+beforeAll(async (): Promise<void> => {
+  postgresPromise = pgPromise();
+  database = postgresPromise(db);
+});
+
+afterAll((): void => {
+  postgresPromise.end();
+});
+
+describe('postgres articles #integration', (): void => {
+  beforeEach(async (): Promise<void> => {
```

Comment:
I think this can exist outside the `describe`, in which case they're all together.